xapian-bindings/go/Makefile.am

```diff
@@ -0,0 +1,38 @@
+GOSRCDIR = ${GOROOT}/src/xapian
+TESTS = ./test
+AM_TESTS_ENVIRONMENT = \
+	abs_builddir='$(abs_builddir)' ;\
+	srcdir='$(srcdir)'
+	export abs_builddir ;\
+	export srcdir ;
+LOG_COMPILER = ${GO} test ./test
+
+install-data-hook:
+	cd ${GOROOT}/src/xapian
+	${GO} install -i -x -a
+
+all : modify
+	cd ${GOROOT}/src/xapian
+	${GO} build -i -x -a
+
+modify : build
+	mkdir -p ${GOSRCDIR}
+	cp -r * ${GOSRCDIR}
+
+build :
+	${SWIG} ${SWIG_FLAGS} -c++ -go -cgo -intgosize 64 go.i
+	awk 'NR==18{print "#cgo CXXFLAGS: ${XAPIAN_CXXFLAGS}"}1' xapian.go > xapian2.go
+	mv xapian2.go xapian.go
+	awk 'NR==19{print "#cgo CXXFLAGS: -std=gnu++11"}1' xapian.go > xapian2.go
```

Comment:
This option isn't supported by all C++ compilers.  You should use the value of `CXXFLAGS` which is set in the Makefile (to which configure will have added any option needed to enable C++11 support).